src/Components/SingleBookView/SingleBookView.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const SingleBookView = ({ trendingBooks, coverImages, descriptions, singleBookId }) => {
+
+    const foundBook = trendingBooks.find(book => book.key === `/works/${singleBookId}`)
```

Comment:
This logic can probably be done up in the parent, so that this component only needs to know about one trending book!